selenium_tests/course_info/course_info_search_tests.py

```diff
@@ -14,3 +14,69 @@ def test_course_search(self):
             year=course['year'], search_term=course['cid'])
 
         self.assertTrue(self.search_page.is_course_displayed(cid=course['cid']))
+
+    def test_course_code_is_display_when_registrar_code_display_is_null(self):
+        """
+        This test will verify that if the registrar_code_display in the
+        database is null, that the user will see the registrar_code
+        that was entered
+        """
+
+        course = self.test_settings['test_course_with_null']
+
+        self.search_for_course(
+            school=course['school'],
+            search_term=course['cid'])
+
+        self.assertTrue(self.search_page.is_course_code_visible(
```

Comment:
I'm not entirely sure these tests do exactly what we're purporting they do; they also seem a bit redundant. There are four tests here, but it seems like we only need two:
- Does the display in the course code column match (exactly) the registrar_code_display for courses that have that field defined? 
- Does the display in the course code column match (exactly) the registrar_code for courses that do not have registrar_code_display defined?

The tests don't seem to be doing that. For one thing, the matches are 'contains', so they're not exact, can we fix that so we're sure we're seeing exactly what we want? It would be great, too, if we could assert that the course code is found in the right td element, instead of just searching for it in the table, in case e.g. the course details column matches it too (although that's unlikely, so just searching for it in the table is probably fine). Finally, I think the four tests can be collapsed into two.